tests/php/Forms/HTMLEditor/HTMLEditorConfigTest.php

```diff
@@ -22,149 +18,288 @@ protected function setUp(): void
     {
         parent::setUp();
 
-        TinyMCEConfig::config()->set('base_dir', 'test/thirdparty/tinymce');
-    }
-
-    public function testEnablePluginsByString()
-    {
-        $c = new TinyMCEConfig();
-        $c->enablePlugins('plugin1');
-        $this->assertContains('plugin1', array_keys($c->getPlugins() ?? []));
-    }
-
-    public function testEnablePluginsByArray()
-    {
-        $c = new TinyMCEConfig();
-        $c->enablePlugins(['plugin1', 'plugin2']);
-        $this->assertContains('plugin1', array_keys($c->getPlugins() ?? []));
-        $this->assertContains('plugin2', array_keys($c->getPlugins() ?? []));
-    }
-
-    public function testEnablePluginsByMultipleStringParameters()
-    {
-        $c = new TinyMCEConfig();
-        $c->enablePlugins('plugin1', 'plugin2');
-        $this->assertContains('plugin1', array_keys($c->getPlugins() ?? []));
-        $this->assertContains('plugin2', array_keys($c->getPlugins() ?? []));
-    }
-
-    public function testEnablePluginsByArrayWithPaths()
-    {
-        // Disable nonces
-        $urlGenerator = new SimpleResourceURLGenerator();
-        Injector::inst()->registerService($urlGenerator, ResourceURLGenerator::class);
-
-        Config::modify()->set(Director::class, 'alternate_base_url', 'http://mysite.com/subdir');
-        $c = new TinyMCEConfig();
-        $c->setTheme('modern');
-        $c->setOption('language', 'es');
-        $c->disablePlugins('table', 'emoticons', 'code', 'link', 'importcss', 'lists', 'autolink', 'searchreplace', 'visualblocks', 'wordcount');
-        $c->enablePlugins(
-            [
-                'plugin1' => 'mypath/plugin1.js',
-                'plugin2' => '/anotherbase/mypath/plugin2.js',
-                'plugin3' => 'https://www.google.com/plugin.js',
-                'plugin4' => null,
-                'plugin5' => null,
+        // Make sure we're using the TextAreaConfig even if a module provides an alternative.
```

Comment:
We're using a NullEditorConfig below, not a TextAreaConfig.  However we probably don't need the NullEditorConfig? I'm not sure why we just don't use the TextAreaConfig here

Reply:
I think there were some additional test cases I was going to add that use both, or something? I thought I had done it, I'll see if I can remember and add - but will remove the extra class if I can't remember.

Reply:
Added the test that uses the `NullEditorConfig` - but updated the other tests to just use `TextAreaConfig`.